app/assets/javascripts/admin/addon/components/admin-config-areas/color-palette.gjs

```diff
@@ -216,6 +269,7 @@ export default class AdminConfigAreasColorPalette extends Component {
                   </span>
                 {{/if}}
                 <form.Submit
+                  @disabled={{this.saving}}
```

Comment:
```suggestion
                  @isLoading={{this.saving}}
```

Seems like form-kit already supports this so no need to add another `@disabled` attribute.